app/src/main/java/com/framgia/wsm/screen/requestleave/RequestLeaveViewModel.java

```diff
@@ -482,4 +1143,16 @@ public void setCompensationToTime(String compensationToTime) {
     public User getUser() {
         return mUser;
     }
+
+    @IntDef({
+            CurrentTimeSelected.NONE, CurrentTimeSelected.CHECK_IN, CurrentTimeSelected.CHECK_OUT,
+            CurrentTimeSelected.COMPENSATION_FROM, CurrentTimeSelected.COMPENSATION_TO
+    })
+    @interface CurrentTimeSelected {
+        int NONE = 0;
+        int CHECK_IN = 1;
+        int CHECK_OUT = 2;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.